seeds/dex/trades/dex_trades_seed.csv

```diff
@@ -197,6 +197,8 @@ polygon,apeswap,1,2022-11-22,0x3d56ef902984864931d8feb5f1e91cf896e2d1d535a4ada39
 polygon,apeswap,1,2022-11-22,0x740f53bcc15bb3a96c1435003e837677f8bc8f0d94d0c3ff6c33f45443cb746b,134,0xc004e2318722ea2b15499d6375905d75ee5390b8,22416.52442997,0x2791bca1f2de4661ed88a30c99a7a9449aa84174,21
 polygon,apeswap,1,2022-11-22,0x194d468d6c62a717806b5ee4b71131e9f834d724cbd4309a525fac5d9a320f64,185,0x0d500b1d8e8ef31e21c99d1db9a6444d3adf1270,0.99267586734214204,0x5d47baba0d66083c52009271faf3f50dcc01023c,12.616273336186259757
 polygon,apeswap,1,2022-11-22,0xe1d484c6453e2f701a0b246764d4c6dac27d99588b4df3ba6caee50f72f21429,290,0x2791bca1f2de4661ed88a30c99a7a9449aa84174,33,0xc004e2318722ea2b15499d6375905d75ee5390b8,35313.94136592
+avalanche_c,yield_yak,1,2023-01-17,0x84c005d0dba60131a638a0eaafee57bdcad502bf04c17ddc7f49551dd20a8135,16,0xa7d7079b0fead91f3e65f86e8915cb59c1a4c664,25001.697591,0xa7d7079b0fead91f3e65f86e8915cb59c1a4c664,25000
```

Comment:
i think we should initiate a new seed file for `dex_aggregator` spells. this approach was built after cow initiated the sector spell, so there is no data for it here. it can be same directory, with updated file name. i'd imagine the data within could be v similar as well, maybe incorporate trace_address in joins though?

thinking out loud, we'd reuse `check_seed_macro`, maybe have a new `check_dex_aggregator_seed` file under tests/generic/ & edit there to potentially add more matching columns or whatever else is needed for aggregators

Reply:
@jeff-dude Alrighty, I'll set one up for the yield yak pr so once this gets merged, I'll update the rest. 

Will also incorporate trace address in the joins, btw when you say that wdym? 


There are call tables I could use to get the trace address so I'd join that with the event table, but curious if you're referring to something else.

Reply:
created the aggregator test / seed. And added trace address from the calls table.

Reply:
@jeff-dude I added trace_address from the call tables but dex_aggregator_trades can't run because cow protocol is using '' as trace address. 

Should I cast the trace_address in my pr to string or use array('') in cow protocol's model? 

Happy to work with either.

Reply:
All the data needed are in the event tables and there would be no need to join the base tables.

Yeah that sounds good to me. Should I go ahead and push a pr and leave the trace address as null ?

Reply:
for yield yak*

Reply:
yeah, you can revert back to not use call tables / trace address on this one and force it null. i'll look to make the trace address updates when i push 1inch

Reply:
okaie ser, will update now.

Reply:
updated ser

Reply:
bump ser @jeff-dude 👀